src/Enums/EventSubType.php

```diff
@@ -30,9 +30,21 @@ class EventSubType
 
     // Triggers whenever a viewer subscribes to a broadcaster's channel.
     public const CHANNEL_SUBSCRIBE = 'channel.subscribe';
-
-    // Triggers whenever a viewer unsubscribes from a broadcaster's channel.
-    public const CHANNEL_UNSUBSCRIBE = 'channel.unsubscribe';
+    
+    /**
+     * @deprecated Please use the CHANNEL_SUBSCRIPTION_END constant
+     */
+    // Triggers whenever a subscription to the specified channel ends.
```

Comment:
The PHPDoc will otherwise not be detected by IDEs

```suggestion
    /**
     * Triggers whenever a subscription to the specified channel ends.
     *
     * @deprecated Please use the CHANNEL_SUBSCRIPTION_END constant
     */
```

Reply:
I simply removed the comment, as the constant should no longer be used